heat-core-utils/src/main/java/com/hotels/heat/core/environment/EnvironmentHandler.java

```diff
@@ -155,6 +155,10 @@ public void setPh(PropertyHandler ph) {
         this.ph = ph;
     }
 
+    public PropertyHandler getPh() {
```

Comment:
Could, the method name, be more descriptive?

Reply:
Renamed to getPropertyHandler().